src/services/game-logic/Sound.ts

```diff
@@ -0,0 +1,24 @@
+class Sound {
+    private lastPlayedMoveSound = 0;
+    private AMOUNT_OF_NORMAL_MOVE_SOUNDS = 3;
+
+    public playCapturingMoveSound(): void {
+        const sound = new Audio('/static/sounds/capture.mp3');
```

Comment:
Wydaje mi się że tu jest problem który słyszała Asia, za każdym razem wczytuje to audio z assetów, zamiast tego możemy wczytać to raz w konstruktorze przechować w ramach this.captureSound i tylko callować play wewnątrz tej metody